uint32 id
float32 speed
---
bool success
string message
